app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/apps/ui/TrackingProtectionAppsAdapter.kt

```diff
@@ -109,66 +107,66 @@ interface AppProtectionListener {
     )
 }
 
-class TrackingProtectionAppViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val context = itemView.context
+class TrackingProtectionAppViewHolder(val binding: RowExclusionListAppBinding) : RecyclerView.ViewHolder(binding.root) {
+    private val context = binding.root.context
 
     fun bind(
         isListEnabled: Boolean,
         excludedAppInfo: TrackingProtectionAppInfo,
         position: Int,
         listener: AppProtectionListener,
     ) {
-        val appIcon = itemView.context.packageManager.safeGetApplicationIcon(excludedAppInfo.packageName)
-        itemView.deviceShieldAppEntryIcon.setImageDrawable(appIcon)
-        itemView.deviceShieldAppEntryName.text = excludedAppInfo.name
+        val appIcon = context.packageManager.safeGetApplicationIcon(excludedAppInfo.packageName)
+        binding.deviceShieldAppEntryIcon.setImageDrawable(appIcon)
+        binding.deviceShieldAppEntryName.text = excludedAppInfo.name
 
         if (excludedAppInfo.isProblematic()) {
             if (excludedAppInfo.isExcluded) {
-                itemView.deviceShieldAppExclusionReason.text =
+                binding.deviceShieldAppExclusionReason.text =
                     getAppExcludingReasonText(itemView.context, excludedAppInfo.knownProblem)
```

Comment:
NIT: I know it's not part of migrating to viewBinding. But because we already have the context in a private variable we could use it instead accessing the `itemView.context`
```suggestion
                    getAppExcludingReasonText(context, excludedAppInfo.knownProblem)
```